### PR TITLE
JBIDE-25833 New OpenShift ITests DeployVariousDockerImagesTest

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by jkopriva** [comment]
green build: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/openshift.itests/362/

**Comment by jkopriva** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
@jkopriva would you rebase the branch, please?

**Comment by jkopriva** [comment]
@odockal done, can you please review/merge?

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/integration/docker/AbstractDockerImageTest.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+	private static final String connectionsURI = System.getProperty("openshift.server").replaceAll("\\:[0-9]*$",
+			":2376");
+	private static final String pathToCertificate = System.getProperty("user.home") + "/.minishift/certs";
+	protected static String DOCKER_CONNECTION = connectionsURI;
+
+	public static void createDockerConnection() {
+		dockerExplorer = new DockerExplorerView();
+		dockerExplorer.open();
+		List<String> connectionsNames = dockerExplorer.getDockerConnectionNames();
+		if (!connectionsNames.isEmpty()) {
```
Comment (odockal) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/integration/docker/AbstractDockerImageTest.java]: How about using [CleanDockerExplorerRequirement](https://github.com/jbosstools/jbosstools-openshift/blob/master/test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/requirements/CleanDockerExplorerRequirement.java)?

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/integration/docker/DeployVariousDockerImagesTest.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+			connection.getProject(PROJECT_NEXUS).delete();
+		}
+		if (connection.projectExists(PROJECT_SPRING_HELLOWORLD)) {
+			connection.getProject(PROJECT_SPRING_HELLOWORLD).delete();
+		}
+	}
+
+	@Before
+	public void closeBefore() {
+		closeBrowser();
```
Comment (odockal) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/integration/docker/DeployVariousDockerImagesTest.java]: Isn't one of the call of closeBrowser useless?

**Comment by odockal** [comment]
LGTM, +1.

**Comment by jkopriva** [comment]
@odockal thx for merging :)

**Pull request merged**